frontend/src/hooks/use-hypershift-enabled.ts

```diff
@@ -1,21 +1,20 @@
 /* Copyright Contributors to the Open Cluster Management project */
 
 import { useState, useEffect } from 'react'
-import { listMultiClusterEngines } from '../resources'
 import { useRecoilState, useSharedAtoms } from '../shared-recoil'
 
 export const useIsHypershiftEnabled = () => {
   const [isHypershiftEnabled, setIsHypershiftEnabled] = useState<boolean>(false)
-  const { managedClusterAddonsState } = useSharedAtoms()
+  const { managedClusterAddonsState, multiClusterEnginesState } = useSharedAtoms()
```

Comment:
Needed to replace the fetch with recoil state to avoid the card "jumping" at load time between a state of hypershift disabled and enabled